### PR TITLE
fix: add delay to print correct log timestamp

### PR DESCRIPTION
**Pull request by parthasl**
## Context
1. In the Kata setup, timestamp printed in the 2nd or 3rd setup step skew and auto-corrects at the last setup step or 1st user step.
2. Flaky test TestTeardownAbort

## Objective
1. Add a temporary hack of 15-sec delay at the start of the async emitter function to resolve the issue. 15-sec delay will only apply for the launcher log step.
2. Fix flaky test TestTeardownAbort

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by vnugopal2000 — APPROVED** [review]

**Review by vnugopal — APPROVED** [review]

File: screwdriver/emitter.go [hunk 1]
```diff
@@ -63,6 +64,13 @@ func (e *emitter) processPipe() {
 	var line string
 	var readErr error
 
+	// temporary hack - without this delay the datetime is printing incorrectly for kata containers
```
Comment (tkyi) [inline screwdriver/emitter.go:67]:
```suggestion
	// TODO: fix temporary hack - without this delay the datetime is printing incorrectly for kata containers
```

**Comment by parthasl** [comment]
@kumada626 @screwdriver-cd/yahoo-japan We are seeing a weird issue "timestamp printed in the 2nd or 3rd setup step skew and auto-corrects at the last setup step or 1st user step" in our build environment (Kata containers). Wanted to check if you have come across this issue. Attached screenshot for your reference. Appreciate your response

timestamp in scm setup step
image


timestamp reverts in compatibility bookend setup step
image

**Review by tkyi — APPROVED** [review]

**Comment by kumada626** [comment]
@parthasl In our environment, we haven't seen this issue yet. However, we are using kata containers in only a few builds. We are continuing to monitor this issue for occurrence.

**Comment by parthasl** [comment]
Thanks, @kumada626 this change should not impact the launcher behavior, but please do let me know if you see any issues.

**Pull request merged**